src/Nri/Ui/QuestionBox/V4.elm

```diff
@@ -293,10 +293,10 @@ viewPointingTo config blockId measurements =
             , css
                 (case measurements of
                     Just { questionBox } ->
-                        [ Css.height (Css.px (questionBox.element.height + 8)) ]
+                        [ Css.height (Css.px (questionBox.element.height + 8)), Css.fontSize (Css.px 18) ]
 
                     Nothing ->
-                        []
+                        [ Css.fontSize (Css.px 18) ]
```

Comment:
These styles are being added to the "question-box-absolute-positioning-spacer" wrapper. I think it might be tidier to add them to the Balloon container or the Balloon innards instead:
e.g.,
```
Balloon.containerCss [ Css.fontSize (Css.px 18) ] 
```

Or 

```
Balloon.css [ Css.fontSize (Css.px 18) ] 
```

Reply:
(Or add the style to an existing containerCss use -- there are 2 lists already!)

Reply:
Sure, I just did it! I think that I added more styles than needed